### PR TITLE
fix: Markdownlint cleanup for ES

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

File: files/es/web/guide/ajax/index.md [hunk 1]
```diff
@@ -56,8 +56,14 @@ Una introducción a AJAX.
...
 - [Una Simple Librería de Interfaz XMLHTTP](http://xkr.us/code/javascript/XHConn/)
 
-<!---->
-
-- #### Temas relacionados
-
-  - : [HTML](es/HTML) · [XHTML](es/XHTML) · [CSS](es/CSS) · [DOM](es/DOM) · [JavaScript](es/JavaScript) · [XML](es/XML) · [XMLHttpRequest](es/XMLHttpRequest) · [XSLT](es/XSLT) · [DHTML](es/DHTML)
+#### Temas relacionados
+
+- [HTML](es/HTML)
```
Comment (JuanVqz) [inline files/es/web/guide/ajax/index.md:61]:
```suggestion
- [HTML](/es/HTML)
```

**Review by JuanVqz — COMMENTED** [review]
I guess we need to add the slash before the localization

File: files/es/web/guide/ajax/index.md (hunk 1 above)
Comment (nschonni) [reply files/es/web/guide/ajax/index.md:61]: You're right, I think I cleanup up a bunch of those in another PR. I'll do a quick scan to see if there are any more in this folder and push the changes if there are a bunch

Comment (nschonni) [reply files/es/web/guide/ajax/index.md]: Took a pass through and cleaned up a few otheres I found in the ES files

**Review by Graywolf9 — APPROVED** [review]
Thank you so much @nschonni it seems a lot less files, thank you for the file for linting

and thank you for the review @JuanVqz

**Pull request merged**